js/build.js

```diff
@@ -263,6 +280,8 @@ Fliplet.Widget.instance('chat', function (data) {
 
     getContacts(false).then(function () {
       return getConversations();
+    }).catch(function(error) {
```

Comment:
@hcarneiro can you please log the error with `console.warn(error)` ? otherwise we can't debug why it failed.